src/classes/Hotel.js

```diff
@@ -1,24 +1,18 @@
 /* eslint-disable max-len */
-
-import Customer from "./Customer"
+import dayjs from "dayjs";
+//^^^^any file to use on this needs to be imported
 
 class Hotel {
 
   //NEED TO FIX THE CONSTRUCTOR FOR STUFF THAT IS 
   // ROOM OR BOOKING DATA BASED ON HOW IT IS BEING IMPORTED VIA THE FETCH CALLS (0-customerData, 2-roomData, 3-bookingData)
   constructor(currentCustomerData, roomData, bookingData) {  
-    // this.hotels = hotelData;
     this.rooms = roomData;
     this.bookings = bookingData;
     this.currentCustomer = currentCustomerData;
-    this.id = bookingData.id;
```

Comment:
I like how you cleaned up the Hotel constructor by making a property to hold each data set. It looks real clean!